pkg/component/worker/config/config_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2022 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	logsv1 "k8s.io/component-base/logs/api/v1"
+	kubeletv1beta1 "k8s.io/kubelet/config/v1beta1"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestToConfigMapData(t *testing.T) {
+	t.Parallel()
+
+	for _, test := range append(roundtripTests,
+		roundtripTest{"nil_produces_zero", nil, map[string]string{}},
+	) {
+		t.Run(test.name, func(t *testing.T) {
+			test := test
+			t.Parallel()
```

Comment:
Let's avoid using `t.Parallel` in the table tests: https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721